SysId/subsystems/drive.py

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+#
+
+from commands2 import Command, Subsystem
+from commands2.sysid import SysIdRoutine
+from wpilib.sysid import SysIdRoutineLog
+
+from wpilib import Encoder, PWMSparkMax, RobotController
+from wpilib.drive import DifferentialDrive
+
+from wpimath.units import volts
+
+from constants import DriveConstants
+
+from typing import Callable
+
+
+class Drive(Subsystem):
+    def __init__(self) -> None:
+        # The motors on the left side of the drive
+        self.left_motor = PWMSparkMax(DriveConstants.kLeftMotor1Port)
+        # TODO this has been disabled because the method is broken. see https://github.com/robotpy/mostrobotpy/issues/64
+        # self.left_motor.addFollower(PWMSparkMax(DriveConstants.kLeftMotor2Port))
+
+        # The motors on the right side of the drive
+        self.right_motor = PWMSparkMax(DriveConstants.kRightMotor1Port)
+        # TODO this has been disabled because the method is broken. see https://github.com/robotpy/mostrobotpy/issues/64
+        # self.right_motor.addFollower(PWMSparkMax(DriveConstants.kRightMotor2Port))
```

Comment:
`addFollower` is fixed.
```suggestion
        self.left_motor.addFollower(PWMSparkMax(DriveConstants.kLeftMotor2Port))

        # The motors on the right side of the drive
        self.right_motor = PWMSparkMax(DriveConstants.kRightMotor1Port)
        self.right_motor.addFollower(PWMSparkMax(DriveConstants.kRightMotor2Port))
```